bin/create-test-release.sh

```diff
@@ -0,0 +1,132 @@
+			exit 1
+		fi
+
+		unzip .cache/woorelease.zip -d .cache >/dev/null
+		rm .cache/woorelease.zip
+	fi
+
+	# Check for Woorelease config file
+	if [[ ! -f ~/.woorelease/config ]]; then
+	 	echo 'Woorelease config file not found (~/.woorelease/config).'
+		echo 'Please follow the configuration steps here: https://github.com/woocommerce/woorelease#prerequisites-for-configuration'
+	 	exit 1
+	fi
+}
+
+print_summary() {
+	echo ''
+	echo 'This script will perform the following actions:'
+	echo "  1. Checkout the branch 'develop' and pull the latest changes"
+	echo "  2. Create a new branch named 'release/$VERSION' using 'develop' as base, and push it to GitHub"
+	echo "  3. Create a new tag named '$VERSION-test', and push it to GitHub."
+	echo "  4. Run Woorelease simulation locally with version=$VERSION (using the newly created branch)"
+}
+
+create_git_branch() {
+	# Create GitHub branch and tag for the test version
+	echo ''
+	echo "> git checkout develop && git pull"
+	git checkout develop && git pull
+	[ $? -eq 0 ] || exit 1
+
+	echo ''
+	echo "> git checkout -b release/$VERSION"
+	git checkout -b release/$VERSION
+	[ $? -eq 0 ] || exit 1
+
+	echo ''
+	echo "> git push origin release/$VERSION"
+	git push origin release/$VERSION
+	[ $? -eq 0 ] || exit 1
+
+	echo ''
+	echo "> git tag $VERSION-test && git push origin $VERSION-test"
+	git tag $VERSION-test && git push origin $VERSION-test
+	[ $? -eq 0 ] || exit 1
+}
+
+create_test_package() {
+	# We simulate creating the release with the version number VERSION (ex: 5.3.0) and not the current test version (ex: 5.3.0-test)
+	echo ''
+	echo "> php .cache/woorelease.phar simulate --product_version=$VERSION https://github.com/woocommerce/woocommerce-gateway-stripe/tree/release/$VERSION"
+	output=$(php .cache/woorelease.phar simulate --product_version=$VERSION https://github.com/woocommerce/woocommerce-gateway-stripe/tree/release/$VERSION | tee /dev/stderr )
+
+	# Update the 'Version' tag in woocommerce-gateway-stripe.php to the current version + '-test'
+	ZIP_FILE=$(echo $output | sed -n "s/.*Skipping upload of asset \(.*\) to GH release.*/\1/p")
+	[ $? -eq 0 ] || exit 1
+
+	cd $(echo "$ZIP_FILE" | rev | cut -d '/' -f3- | rev)
+	sed -i '' "s/^ \* Version: .*$/ * Version: $VERSION-test/" woocommerce-gateway-stripe/woocommerce-gateway-stripe.php
+	zip --update woocommerce-gateway-stripe/woocommerce-gateway-stripe.zip woocommerce-gateway-stripe/woocommerce-gateway-stripe.php
+	cd -
+	cp $ZIP_FILE .
+}
+
+# -----
+
+check_prerequisites;
+print_summary;
+echo ''
+read -p "Do you want to continue? (yes/no) " -n 1 -r
+echo ''
+if [[ ! $REPLY =~ ^[Yy]$ ]]; then
+	exit 0
+fi
+
+create_git_branch;
+create_test_package;
+
+echo ''
+echo "Test release $VERSION generated successfully."
+echo ''
+echo "You need to upload the zip package 'woocommerce-gateway-stripe.zip' to the tag '$VERSION-test' on GitHub (creating a version and marking it as pre-release BEFORE saving)."
```

Comment:
I wonder if we could create the release and upload the zip file via the script here 🤔 

That'd be pretty sweet 👌

Reply:
Copied the release creation and asset upload form the WC Pay repo... and also added manual instructions as a fallback